easybuild/easyconfigs/v/Visit/Visit-2.12.0-CrayGNU-2016.11.eb

```diff
@@ -0,0 +1,103 @@
+# contributed by Victor Holanda Rusu (CSCS)
+#            and Jean Favre (CSCS)
+#
+easyblock = 'CMakeMake'
+
+name = 'Visit'
+version = "2.12.0"
+
+homepage = 'https://wci.llnl.gov/codes/visit/'
+description = """VisIt is an Open Source, interactive, scalable, visualization,
+animation and analysis tool."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'usempi': True, 'verbose': False}
+
+#
+# Visit downloads all its dependencies by itself during the configuration.
+# The downloads may fail due to different reasons. So, we have pre-packaged
+# visit with the downloaded dependencies.
+#
+# Below you will find the steps to create the SOURCELOWER_TAR_GZ file used in
+# this recipe.
+#
+# wget http://portal.nersc.gov/project/visit/releases/2.12.0/visit2.12.0.tar.gz
+# tar xfz visit2.12.0.tar.gz
+# cd visit2.12.0
+# ./src/svn_bin/build_visit --console --no-visit --system-cmake --boost  \
+#                           --h5part --no-pyside --system-python --vtk \
+#                           --mesa --parallel --hdf5 --silo --szip \
+#                           --makeflags -j20 --netcdf --qt --download-only
+# cd ..
+# rm -rf visit2.12.0.tar.gz
+# tar cfz visit-2.12.0.tar.gz visit2.12.0
+# mkdir -p $EASYBUILD_SOURCEPATH/v/Visit
+# cp visit-2.12.0.tar.gz $EASYBUILD_SOURCEPATH/v/Visit/
+#
+# NOTES:
+# * The new visit package is named visit-2.12.0.tar.gz, not visit2.12.0.tar.gz
+#   as the original file.
+# * The download-only option must not be included in the actual build.
+# * The configuration options are the same as the ones below. Except for the
+#   boost and download-only options. In our case, boost is already provided by
+#   a module. So, we can choose to use the module or have visit build it.
+#   Here, we are using the module.
+# * We are using the system-python because one cannot compile pyhton with xalt
+#   enabled.
+# * In the preconfigopts commands below, there is an additional <<< "yes".
+#   This is used to accept the Qt License. This is not needed when using
+#   --download-only
+#
+
+# In order to use the original version uncomment these lines
+#source_urls = ['http://portal.nersc.gov/project/visit/releases/%(version)s' ]
+#sources = ['visit2.12.0.tar.gz']
+
+# In order to use the pre-downloaded version use the source below
+sources = [SOURCELOWER_TAR_GZ]
+
+srcdir='../src'
+preconfigopts  = ' export PAR_COMPILER=`which cc`; '
```

Comment:
Do you really need the full path to the cc wrapper? 

Normally we should use the variables defined by easybuild for specifying the compilers:
```
  export CC="cc"
  export CXX="CC"
  export F77="ftn"
  export F90="ftn"
```